content/docs/reactivesearch/react-searchbox/quickstart.md

```diff
@@ -11,39 +11,342 @@ sidebar: 'docs'
 nestedSidebar: 'react-searchbox-reactivesearch'
 ---
 
-[react-searchbox](https://github.com/appbaseio/react-searchbox) - It offers a lightweight (~30KB: Minified + Gzipped) and performance focused searchbox UI component to query and display results from your Elasticsearch app (aka index) using declarative props. It is an alternative to using the [DataSearch component](/docs/reactivesearch/v3/search/datasearch/) from ReactiveSearch.
+[react-searchbox](https://github.com/appbaseio/react-searchbox) - React SearchBox is a standalone, lightweight search library which allows you to add custom UI components and allow seamless search systems.
```

Comment:
```suggestion
[react-searchbox](https://github.com/appbaseio/react-searchbox) - React Searchbox is a lightweight search library (30KB gzipped size) for quickly creating search experiences powered by Elasticsearch. It takes care of querying to Elasticsearch, managing search state and updating your search UI. Searchbox is built with ease of integration with any design system in mind.
```

Reply:
I've edited this. See if this sounds better. We can make changes to this for better accuracy.